requirements.txt

```diff
@@ -1,44 +1,10 @@
-# tensor search:
+# This is a reduced set of requirements used for Marqo's tensor search module
+# If you are running this on your local machine you will need to install additional
+# requirements specified here: https://github.com/marqo-ai/marqo-base/blob/main/requirements.txt
```

Comment:
This will be a bit difficult to manage. It might confuse the IDE (I have tested though). Also when checking out an old commit we don't really know which `marqo-base` commit corresponds to this for the right version.

How about including a requirements.dev.txt in the repo with all dependencies listed?

Reply:
Good point. Added the new file and updated dev README

Reply:
Also, keep in mind that if you needed to, you can trace the marqo-base version via the marqo-base number (but it isn't super straightforward)